app/src/main/java/com/duycuong/weather/utils/TimeUtils.java

```diff
@@ -1,6 +1,7 @@
 package com.duycuong.weather.utils;
 
 import java.text.SimpleDateFormat;
+import java.util.Date;

```

Comment:
ERROR: Unused import: java.util.Date. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports